content/workspace/openbb-copilot.md

```diff
@@ -12,336 +12,616 @@ keywords:
-discussions, ask follow-up questions, and refine your queries based on previous
-responses. The conversation history helps OpenBB Copilot understand the context of
-your questions and provide more relevant and personalized assistance throughout
-your interaction.
+This flexibility allows seamless transitions between quick consultations and deep analytical sessions without losing context or interrupting your workflow.
 
-For example, if you ask about the price-to-earnings (P/E) ratio, and then follow
-up by asking about other important financial ratios, OpenBB Copilot will
-understand the context and suggest relevant ratios:
+And of course you are capable of hiding the AI agent if only utilizing the workspace.
 
-<div style={{display: 'flex', justifyContent: 'center'}}>
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/ada_chat_history.png" alt="chat history openbb copilot" width="60%" height="60%" />
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/9e9a76dc-7e4a-4e04-9211-086ee147c6c1.png"
+    width="100%"
+  />
 </div>
 
-If you'd like to clear the history of the current conversation, you can do so by
-clicking on the trashcan icon in the upper right corner of the OpenBB Copilot
-window. It is **highly recommended** to clear the chat history when you have a
-new question that is unrelated to your current conversation, to prevent
-confusing OpenBB Copilot:
+### Prompt library
+
+The integrated prompt library serves as a productivity multiplier by storing and organizing your most valuable queries. You can save complex prompts and even tag widgets, to ensure the right context is utilized for the right prompt.
 
-<div style={{display: 'flex', justifyContent: 'center'}}>
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/ada_trash_can_icon.png" alt="clear chat button openbb copilot" width="100%" height="100%" />
+This feature ensures consistency across team members and dramatically reduces the time spent retyping similar requests.
+
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/f145bb2d-89bf-49f9-8fd7-af24d78e4dff.png"
+    width="100%"
+  />
 </div>
 
-## General question answering
+### Model
+
+The OpenBB Copilot leverages cutting-edge large language models from OpenAI, specifically optimized for financial analysis and data interpretation tasks.
 
-The simplest functionality of OpenBB Copilot is its general question answering
-capability. For example, you can ask Copilot things like:
+For enterprise deployments, OpenBB provides seamless integration with your organization's Azure OpenAI or dedicated OpenAI deployments, ensuring data sovereignty, compliance with internal security policies, and the ability to customize model behavior according to your firm's specific requirements and risk parameters.
```

Comment:
This is a very long sentence. Shall we split it into 2?  Nothing wrong with the content, it's just a bit heavy.